project_deadline/views/project_project_view.xml

```diff
@@ -9,14 +9,23 @@
         <field name="arch" type="xml">
             <xpath expr="//span[@t-if='record.partner_id.value']" position="after">
                 <t
-                    t-if="record.date.raw_value and record.date.raw_value lt (new Date())"
+                    t-if="record.date.raw_value and record.date.raw_value lt (new Date().toISOString())"
                     t-set="red"
                 >oe_kanban_text_red</t>
```

Comment:
**suggestion** (non-blocking): Instead of comparing two strings, we could turn `record.date.raw_value` into a Date object. See here for an example:

https://github.com/OCA/social/blob/16.0/mail_activity_board/views/mail_activity_view.xml#L186-L190

Reply:
resolved